### DISCUSSION TITLE
Just a question. how to see a suggestion of users in "Panels -  Value"?

### DISCUSSION THREAD

**Discussion by StasSpawn** [dbeaver/dbeaver#19707] [Q&A]
I'm watching this video 
https://youtu.be/SvPZ0DN8y4o?t=744

And here when he click un NULL user - Value panel showed him created user. 
![2023-04-06_205231](https://user-images.githubusercontent.com/56681850/230457974-c1a86bb5-93ec-4261-b3b8-02f76ac04062.png)

I did the same code, press NULL user and my Value panel is empty. 

Is there a button to show suggestions?

**Comment by E1izabeth** [comment]
It works if you have a foreign key on this column to another table.

**Comment by E1izabeth** [comment]
Maybe you haven't updated metadata after foreign key creation? Try pressing F5 on the connection in navigator

**Comment by StasSpawn** [comment]
F5 doesn't help. 

And I'm new to databases. Don't know what metadata here means.
